package.json

```diff
@@ -8,18 +8,19 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@react-google-maps/api": "^2.7.0",
```

Comment:
ok technically this is outside the PR scope but we're going to commit it soon anyway and otherwise I'd need to regenerate all the packages over again and potentially cause merge conflicts tomorrow, plz forgive